debug-tools/Cargo.toml

```diff
@@ -7,5 +7,5 @@ publish = false
 
 [dependencies]
 framework = { path = "../framework" }
-embedded-graphics = "0.7.0-beta.1"
+embedded-graphics = "0.7.0-beta.2"
```

Comment:
This isn't really necessary, because of the `patch` section in the root `Cargo.toml` (but doesn't hurt either). Using git dependencies everywhere had previously caused some issues (which I don't remember in detail at the moment).